### PR TITLE
implemented feedback link

### PR DESCRIPTION
**Pull request by afrzhussain**

### CODE REVIEW

File: src/components/btn_event_feedback_link.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react'
+
+
+
+const FeedbackLink = ({link}) => (
+  <div className="float">
+    <a href={link}
```
Comment (haxzie-xx) [inline src/components/btn_event_feedback_link.js]: Somehow this doesn't get rendered properly, the rendered button on the page doesn't seem to have the `href` attribute, thus it's not clickable. I think you might wanna investigate this further.

File: src/templates/event.js [hunk 2]
```diff
@@ -14,13 +16,41 @@ export default function EventTemplate({ data }) {
...
+   function registration_link(){
+        console.log(post.frontmatter.link)
+        if(post.frontmatter.link !== undefined && post.frontmatter.link !== null) {
+        if (post.frontmatter.link.length > 0) {
         return <RegistrationLink link={post.frontmatter.link} />
       }
     }
+
   }
+ function feedbac_link(){
```
Comment (haxzie-xx) [inline src/templates/event.js]: Can we rename this function to `feedback_link`?

**Review by haxzie-xx — CHANGES_REQUESTED** [review]
I have left a few pointers, IWhat do you think about showing the feedback link only for a week after the event date?

**Comment by haxzie-xx** [comment]
@coderhawk999 I have added a few changes and submitted a PR to your fork, take those changes and you are good to continue further.

**Comment by afrzhussain** [comment]
Okay cool , il start working on the changes :+1:

**Comment by afrzhussain** [comment]
@haxzie  i have made the requireed changes..

**Review by haxzie-xx — APPROVED** [review]

**Comment by haxzie-xx** [comment]
Awesome :+1: Good job! There are a few additional things to be taken care of now. Since we have mentioned the no. of days the feedback link will be available directly as hardcoded value, we gotta change it. It's a good practice to move things like these into a separate configuration file. you can have a look at the `gatsby-config.js file` in the root directory which sports some of the most used strings. We can create something similar to store these configuration options and `import()` the contents wherever it's required.

**Comment by haxzie-xx** [comment]
@coderhawk999 any updates on this?

**Comment by afrzhussain** [comment]
Dint understand how to put the date class to json data . If give an example . Would be great  
@haxzie

**Comment by afrzhussain** [comment]
@haxzie Code review :clap::clap:

**Comment by haxzie-xx** [comment]
> Dint understand how to put the date class to json data . If give an example . Would be great
> @haxzie

We just need to create a `config.json` or `config.js` to store the no. of Days the FeedBack link should be visible. It can be an integer and we can request import the values and check it in here.
https://github.com/haxzie/sosc-website/blob/0a4dcf4e692de4d66b72200911e58205ec4935d5/src/templates/event.js#L23

**Pull request merged**